Sources/Crossroad/Router.swift

```diff
@@ -73,6 +73,12 @@ public final class Router<UserInfo> {
                 continue
             }
             let route = Route(pattern: patternURL, handler: handler)
+            if self.routes.contains(where: { element in
```

Comment:
Please check duplication on another `register` method.
https://github.com/giginet/Crossroad/pull/32/files#diff-16d3d627409dd0e34b5e3c2cd6dd0af0dbeb7e6a6c5b4e9c5d025b08d0fdc5f0R30

This implementation depends on another one.

Reply:
Fixed https://github.com/giginet/Crossroad/pull/32/commits/e76c65e7fd0634b63235742272da47f7fba3edcc